content/docs/usage/setup/access.mdx

```diff
@@ -2,4 +2,51 @@
 title: Getting Access
 ---
 
-Coming Soon.
+## T6 Hardware
+### With a monitor and keyboard available
+
+- Insert the microSD card with image flashed.
+- Ensure you have an ethernet cable plugged into your T6 box.
+- Plug in a HDMI screen, make sure it's plugged into HDMI1.
+- Plug in a keyboard.
+- Plug power in.
+- Wait until the box finishes booting, this could take anywhere from 2 to 10 minutes.
+- Once finished, login with the username `shibe` and the password `suchpass`. You will be prompted to change the password.
+- Enter command `ip addr` and take note of your IP address.
+- Ignore the `lo: <LOOPBACK>` section, look for another one.
+- Look for a line that starts with inet followed by your IP address, in this example `192.168.1.5`. It may instead start with `10.` or `192.168` or `172.`.
+- See the screenshot below.
+- In a web browser, visit http://your_ip_address:8080 to start configuring your Dogebox.
+- In this example, it would be http://192.168.1.5:8080 but your IP address will be different.
+- This will load the Dogebox Setup page.
+
+### Without a monitor and keyboard available
+- Insert the microSD card with image flashed.
+- Ensure you have an ethernet cable plugged into your T6 box.
+- Plug power in.
+- Wait until the box finishes booting, this could take anywhere from 2 to 10 minutes.
+- In a web browser, visit https://dogebox.local:8080 to start configuring your Dogebox.
+- This will load the Dogebox Setup page.
+
```

Comment:
We should probably add a step in about following the prompts on screen to install to the eMMC. These steps are common regardless of access method.

Reply:
I suppose the page `content/docs/usage/setup/disk.mdx` are these steps